linode/manager
# refactor: [M3-6282] MUI v5 Migration - `SRC > Features > StackScripts` pt1

## Description 📝
Part 1 of migrating the MUI styling for the `StackScripts` package. This PR is for the `Partials`, `SelectStackScriptPanel`, `StackScriptBase`, `StackScriptCreate` and `StackScriptForm` subpackages.

## Changes  🔄
- Migrated styling from makeStyles to use Styled components and sx 
- Removed React.FC
- updated imports to be named exports instead of default exports (unless lazily exported / the component had higher order functions applied to it)
- removed unnecessary index.tsx files

## How to test 🧪
There should be no visual changes.

### Prerequisites
(How to setup test environment)
- Set up local host and navigate to the stack scripts section 

### Verification steps 
(How to verify changes)
- Verify that there are no visual changes on the StackScript landing page
- Verify that the StackScript create page still looks and functions the same
- Verify that the StackScript section in the Linode Create flow still looks and functions the same

## As an Author I have considered 🤔

*Check all that apply*
